core/src/validator.rs

```diff
@@ -526,11 +529,20 @@ impl Validator {
                 (None, None)
             };
 
+        if !config.no_poh_speed_test {
```

Comment:
I wonder if we'll need to check this periodically (with a less drastic result than exiting).  There's some evidence of thermal throttling that could affect PoH after the validator runs for some time or when load increases

Reply:
We have the metrics to check for now if the user wants to see if it's deviating from expected.